src/app/public/src/modules/tutorial/tutorial.component.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../styles/_variables';
+
+.stache-tutorial {
+  counter-reset: myCounter;
+  display: flex;
+  flex-direction: column;
+  background: $stache-color-gray-lightest;
+  border: 1px solid $sky-color-gray-lighter;;
+  border-radius: 3px;
+  padding: 20px 60px;
+  @media (max-width: $sky-screen-md-min) {
```

Comment:
Default styles need to be mobile-first (breakpoints should almost always be `min-width`).